src/definitions/modules/calendar.less

```diff
@@ -127,6 +127,10 @@
   color: @disabledTextColor;
 }
 
+.ui.calendar .ui.table tr .adjacent:not(.disabled) {
+  color: @disabledTextColor;
```

Comment:
When an adjacent day should be selectable, there must be a clear visual indication about that. So displaying selectable adjacent days in the same color as disabled ones isn't clear to the user. I also think such days should be visualised to indicate they belong to one of the adjacent months.
Therefore i suggest the following change:

calendar.variables:
```less
@adjacentTextColor: @mutedTextColor;
@adjacentBackground: @subtleTransparentBlack;
```

calendar.less:
```less
.ui.calendar .ui.table tr .adjacent:not(.disabled) {
    color: @adjacentTextColor;
    background: @adjacentBackground;
}
```

That will look as follows and even if there are still disabled days in between  (Feb 25th) this is much more understandable to the user, what is selectable and what's not:

![image](https://user-images.githubusercontent.com/18379884/54155734-14912700-4445-11e9-8fd8-9dd703c4a1fd.png)

Reply:
Updated PR: Now selectable adjacent days are visually distinguished.